### PR TITLE
New package: xearth

### PR DESCRIPTION
**Pull request by IDF31**

### CODE REVIEW

File: srcpkgs/xearth/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'xearth'
+pkgname=xearth
+version=1.1
+revision=1
+build_style=gnu-make
+makedepends="libX11-devel libXt-devel"
+short_desc="Xearth sets the X root window to an image of the Earth"
+maintainer="IDF31 <idf31@protonmail.com>"
+license="custom"
```
Comment (maxice8) [inline srcpkgs/xearth/template]: please provide a text of the license

File: srcpkgs/xearth/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'xearth'
+pkgname=xearth
+version=1.1
+revision=1
+build_style=gnu-make
```
Comment (maxice8) [inline srcpkgs/xearth/template]: this build_style doesn't exist, gnu-makefile

File: srcpkgs/xearth/template (hunk 1 above)
Comment (IDF31) [reply srcpkgs/xearth/template]: Do I provide the text in license variable?

**Comment by maxice8** [comment]
On Tue, Aug 21, 2018 at 06:21:17PM +0000, Iorga Dragos Florian wrote:
> IDF31 commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,22 @@
> +# Template file for 'xearth'
> +pkgname=xearth
> +version=1.1
> +revision=1
> +build_style=gnu-make
> +makedepends="libX11-devel libXt-devel"
> +short_desc="Xearth sets the X root window to an image of the Earth"
> +maintainer="IDF31 <idf31@protonmail.com>"
> +license="custom"
> 
> Do I provide the text in license variable?
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2036#discussion_r211709387

No but i want to know which license it is.

**Comment by Johnnynator** [comment]
@maxice8 https://xearth.org/HISTORY Bottom of the file is the Licensing part.
```
Permission to use, copy, modify and freely distribute xearth for
non-commercial and not-for-profit purposes is hereby granted without
fee, provided that both the above copyright notice and this permission
notice appear in all copies and in supporting documentation.
```
I think the `non-commercial and not-for-profit` part makes it nonfree ?

**Comment by maxice8** [comment]
On Tue, Aug 21, 2018 at 11:29:37AM -0700, John Zimmermann wrote:
> @maxice8 https://xearth.org/HISTORY Bottom of the file is the Licensing part.
> ```
> Permission to use, copy, modify and freely distribute xearth for
> non-commercial and not-for-profit purposes is hereby granted without
> fee, provided that both the above copyright notice and this permission
> notice appear in all copies and in supporting documentation.
> ```
> I think the `non-commercial and not-for-profit` part makes it nonfree ?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2036#issuecomment-414776024

Certainly does.

File: srcpkgs/xearth/template (hunk 2 above)
Comment (IDF31) [reply srcpkgs/xearth/template]: Im confused right now, is it `gnu-make` or `gnu-makefile`

**Comment by IDF31** [comment]
I tried with both @Johnnynator's and @maxice8 with `make_build_target` and `make_cmd` but they dont work.

**Comment by Johnnynator** [comment]
@IDF31 Revert your install changes to you old do_install(). The Makfile doesn't provide a Install target.

**Comment by Johnnynator** [comment]
Also please squash every commit into a single one. So there is a clean history in the main  repo once it get merged.

**Comment by IDF31** [comment]
@Johnnynator yeah sorry for that, it's the first time i make a void package, i promise if i will try squashing all my commits into one from now on.
I did the do_install() because in the README it said to copy the bin and the .man manually so that's why i did it in the first place.

**Comment by IDF31** [comment]
I'm going to close this as it gone too far and I don't want to mess too much this repo. This software is obsolete and non free anyways, which I don't see as an improvment. I'm sorry.

**Comment by IDF31** [comment]
I managed to make it work again.

**Pull request closed without merging**